_includes/install/file-system-perms-oneuser_22.md

```diff
@@ -23,7 +23,7 @@ To set permissions before you install the Magento software:
 3.	If you haven't done so already, get the Magento software in one of the following ways:
 
 	*	[Compressed archive]({{ page.baseurl }}/install-gde/prereq/zip_install.html)
-	*	[Composer metapackage]({{ page.baseurl }}/install-gde/prereq/integrator_install.html)
+	*	[Composer metapackage]({{ page.baseurl }}/install-gde/composer.html)
```

Comment:
Possible 2.0 using include? If used will 404 when clicked in 2.0 pages.

Reply:
I searched the 2.0 files for references to this include, but got zero results.